src/pybind/mgr/mirroring/fs/dir_map/state_transition.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from enum import Enum, unique
+from typing import Dict
+
+log = logging.getLogger(__name__)
+
+@unique
+class State(Enum):
+    STATE_UNASSOCIATED   = 0
```

Comment:
nit, no need to add the `STATE_` prefix.

Reply:
OK -- I probably just got used to doing it the C++ way where the enum class specifier is really not required to access an enum.

Reply:
[OT] FWIW, i'd suggest use scoped enum where the members are prefixed by the class name when they are referenced. https://en.cppreference.com/w/cpp/language/enum#Scoped_enumerations

Reply:
thanks @tchaikov